### DISCUSSION TITLE
Your project featured in Svelte developers tools roundup

### DISCUSSION THREAD

**Discussion by SebConejo** [BearToCode/carta#127] [Show and tell]
Hi Davide,

I recently wrote an article featuring Carta among the top 10 essential tools for Svelte developers in 2025.

If you could share it or comment on the post, it would help increase its visibility and bring more attention to these great tools: http://bit.ly/3C1zemq

Best regards,

**Comment by BearToCode** [comment]
Thanks for promoting this project. I added a comment and some reactions 👍
